src/code/z_parameter.c

```diff
@@ -3779,7 +3779,7 @@ void Interface_Draw(GlobalContext* globalCtx) {
 
     if (interfaceCtx->unk_244 != 0) {
         gDPPipeSync(OVERLAY_DISP++);
-        gSPDisplayList(OVERLAY_DISP++, sSetupDL_80125A60);
+        gSPDisplayList(OVERLAY_DISP++, sMaterialDL_80125A60);
```

Comment:
idk if this one qualifies as a "material" dlist since it's used on a textured (2d) rectangle